lib/components/ContentPanel/ContentPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import * as classnames from 'classnames/bind';
+import { ActionTriggerButton } from '../ActionTrigger';
+import { Button } from '../Button';
+
+const cx = classnames.bind(require('./ContentPanel.module.scss'));
+
+
+export interface ContentPanelProperties {
+    title: string;
+    content: React.ReactNode | string;
+    actions: {
+        confirm?: {
+            label: string;
+            event: Function;
+        };
+        cancel: {
```

Comment:
nit: should cancel be optional too?

Reply:
no because there is always an X button on the top-right to collapse the panel you want to specify the title for this action